macros/SectionOnPage.ejs

```diff
@@ -5,23 +5,58 @@
 //
 // $0   Page path
 // $1   Section name
+// $2   Element name to wrap the page title (not the section name) in;
+//      leave blank or don't specify this parameter to not wrap the title.
+//      Don't include the angle brackets! Only specific tags are permitted;
+//      they're in the allowedWrappers array. OPTIONAL.
+//
+// Example:
+//
+// {{SectionOnPage("/en-US/docs/Web/API/RTCPeerConnection",
+//                 "RTCSignalingState enum", "code")}}
+
+var allowedWrappers = [
+    "code",
+    "kbd",
+];
 
+let section = $1;
 var lang = env.locale;
 var text = "";
 var page = wiki.getPage($0);
-var title = page.title;
+var title = kuma.htmlEscape(page.title);
+
+var commonLocalStrings = string.deserialize(template("L10n:Common"));
+var localize = mdn.getLocalString;
+
+var summary = (page && page.summary) ? mdn.escapeQuotes(page.summary) :
+    localize(commonLocalStrings, "summary");
+
+if (!title || title == undefined || title == "undefined") {
```

Comment:
I think you want `if (title === undefined)` here. 

`title == "undefined"` looks like you wanted to actually write `typeof title === "undefined"`, which I think isn't needed as the title variable has been declared, so there is no risk of a ReferenceError here.

Reply:
Actually, I do in fact mean title == "undefined" here. Turns out kuma.htmlEscape(page.title) returns the string "undefined" if the page doesn't exist.

Reply:
OMG